src/components/ItemsList.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from 'react';
+import firebase from '../lib/firebase';
```

Comment:
I think you could use the react-firebase-hooks here as it provides listener for docs and collections. You need to do something similar to whats done in FirebaseHooks.js. Its my first time using firebase as well so I am not 100% sure if thats the proper way to do it so I will also add @segdeha to share his thoughts on this.

Reply:
You can check out https://github.com/the-collab-lab/tcl-19-smart-shopping-list/pull/18/commits/b264dcbccaa3ab9f649d0ba543aa8d5840c72848